networking/metallb/metallb-configure-return-traffic.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: ASSEMBLY
+[id="metallb-configure-return-traffic"]
+= Managing symmetric routing with MetalLB
+include::_attributes/common-attributes.adoc[]
+:context: metallb-configure-return-traffic
+
+toc::[]
+
+As a cluster administrator, you can effectively manage traffic for pods behind a MetalLB load-balancer service with multiple host interfaces by implementing features from MetalLB, NMState and OVN-Kubernetes. By combining these features in this context, you can provide symmetric routing, traffic segregation, and support clients on different networks with overlapping CIDR addresses. 
+
+To achieve this functionality, learn how to implement virtual routing and forwarding (VRF) instances with MetalLB, and configure egress services.
+
+:FeatureName: Configuring symmetric traffic by using a VRF instance with MetalLB and an egress service
+include::snippets/technology-preview.adoc[]
+
+[id="challenges-of-managing-symmetric-routing-with-metallb"]
+== Challenges of managing symmetric routing with MetalLB 
+
+When you use MetalLB with multiple host interfaces, MetalLB exposes and announces a service through all available interfaces on the host. This can present challenges relating to network isolation, asymmetric return traffic and overlapping CIDR addresses. 
+
+One option to ensure that return traffic reaches the correct client is to use static routes. However, MetalLB cannot isolate the services and then announce each service through a different interface. Additionally, static routing requires manual configuration and requires maintenance if remote sites are added. 
+
+The default behavior for {product-title} is to assign the IP address of the host network interface as the source IP address for traffic originating from pods. When implementing a MetalLB service, there are scenarios where external systems expect the source and destination IP address for an application to be the same.
+
+[id="overview-of-managing-symmetric-routing-using-vrf-based-networks-with-metallb"]
+== Overview of managing symmetric routing using VRF-based networks with MetalLB
+
+You can overcome the challenges of implementing symmetric routing with MetalLB by using NMState to configure a VRF instance on a host, associating the VRF instance with MetalLB and configuring an egress service for egress traffic with OVN-Kubernetes.
```

Comment:
It would make more sense to state that the VRF is associated with a BGP Peer rather than MetalLB